randovania/games/prime1/logic_database/Chozo Ruins.txt

```diff
@@ -2085,19 +2119,32 @@ Extra - aabb: [397.9558, -8.934963, 4.7323303, 483.05167, 117.17189, 59.302334]
               Can Use Bombs
               Knowledge (Beginner) and Can Use Power Bombs
               All of the following:
-                  # https://youtu.be/bh18bp4makg
+                  # Get to the Door via Out of Bounds
                   Clip Through Objects (Expert) and Single Room Out of Bounds (Advanced) and Standable Terrain (Advanced) and Can Use Boost Ball
                   Any of the following:
+                      # With Space Jump: https://youtu.be/bh18bp4makg?t=47
                       Space Jump Boots
-                      # https://www.youtube.com/watch?v=LdBFCaOFDE4
+                      # Without Space Jump: https://www.youtube.com/watch?v=LdBFCaOFDE4
                       Movement (Expert)
   > Door to Watery Hall Access
-      Any of the following:
-          After Watery Hall Gate
-          Clip Through Objects (Beginner) and Can Use Boost Ball
+      All of the following:
+          Any of the following:
+              # Get through Gate
+              After Watery Hall Gate
+              Clip Through Objects (Beginner) and Can Use Boost Ball
+          Any of the following:
+              # Deal with Vines
+              Space Jump Boots or Normal Damage or Shoot Any Beam
```

Comment:
or expert combat dash

Reply:
Not quite following from where and locking on to what. Got a video of it by chance?

Reply:
https://github.com/user-attachments/assets/c43e7dc2-d712-435a-a0cc-7b2be1c241c8